src/app/services/constants.ts

```diff
@@ -36,6 +36,9 @@ if (document.location.hostname === "localhost") {
     apiPath = "https://isaacscience.eu.ngrok.io/isaac-api/api";
 }
 export const isTest = document.location.hostname.startsWith("test.");
+export const isStaging = document.location.hostname.startsWith("staging.");
```

Comment:
It's not ideal to have dev/test/staging stuff included in production files. Usually we'd try and do this by pulling them from config

Reply:
Where would be the best place to relocate them to? I'm assuming it would be best to put that in as a separate ticket?

Reply:
Ok yeah we already have a .env file and webpack config. I've created a new ticket for this